docs/getting-started/android/screenshots.md

```diff
@@ -125,7 +125,20 @@ You can use [QuickDemo](https://github.com/PSPDFKit-labs/QuickDemo) to clean up
 
 ## Generating Screenshots with Screengrab
 
-- Run `./gradlew assembleDebug assembleAndroidTest` to generate debug and test APKs
+- Run `./gradlew assembleDebug assembleAndroidTest` manually to generate debug and test APKs
+  - You can also create a lane and use `build_android_app`:
+    ```ruby
+    desc "Build debug and test APK for screenshots"
+    lane :buildForScreengrab do
```

Comment:
This shouldn't be camel case, should be `build_for_screengrab`

Reply:
Good point 👍